admin/templates/configuration/etc/sysconfig/qserv

```diff
@@ -50,8 +50,6 @@ export LD_LIBRARY_PATH={{LD_LIBRARY_PATH}}
 # Make sure core file is not discarded
 ulimit -c unlimited
 
-# used by qserv-czar
-export QSW_RESULTDIR=${QSERV_RUN_DIR}/qserv-run/tmp
```

Comment:
It's not used anywhere, I thought I would just remove it as a part of the cleanup.